src/cl/nimbus/nimbus_launcher.star

```diff
@@ -263,17 +281,19 @@ def get_config(
 	if len(extra_params) > 0:
 		cmd.extend([param for param in extra_params])
 
+	node_artifact_uuid = node_keystore_files.files_artifact_uuid if node_keystore_files != None else package_io.NO_ARTIFACT_UUID
+	files = {
+			GENESIS_DATA_MOUNTPOINT_ON_CLIENT: genesis_data.files_artifact_uuid,
+	}
+	if node_artifact_uuid != package_io.NO_ARTIFACT_UUID:
```

Comment:
we don't need the NO_ARTIFACT_UUID at all, we can jsut

if node_keystore_files:
     iles[VALIDATOR_KEYS_MOUNTPOINT_ON_CLIENT] = node_keystore_files.files_artifact_uuid